css/templates/e-commerce/page.css

```diff
@@ -636,6 +655,34 @@ amp-selector [option][selected] {
   border-bottom: 1px solid var(--color-separator);
 }
 
+.commerce-cart-added:after {
+  content: '1';
```

Comment:
When item has been added to the shopping cart, a badge animates in with the number one. It's hard to use basscss classes here since they are all defined in the `after` pseudoclass.